src/controllers/calendars.cr

```diff
@@ -60,7 +60,8 @@ class Calendars < Application
     all_calendars = Set.new((calendars || "").split(',').map(&.strip.downcase).reject(&.empty?))
     all_calendars.concat(candidate_calendars)
     calendars = all_calendars.to_a
-    return [] of Availability if calendars.empty?
+
+    render :not_found, json: "no tenants exist" if calendars.empty?
```

Comment:
This should just be a plain return value
```suggestion
    return "no tenants exist" if calendars.empty?
```

Then you want to define a custom return code for String types
https://spider-gazelle.net/getting_started/routing/#response-codes

This way it's documented properly when you generate OpenAPI docs for the routes